Sources/Logging/Logging.swift

```diff
@@ -17,27 +17,57 @@
     import Glibc
 #endif
 
-/// This is the protocol a custom logger implements.
+/// A `LogHandler` is an implementation of a logging backend. You only need to worry about `LogHandler`s if you wish
```

Comment:
Cut line after the first sentence?

Perhaps avoid "worry" phrasing? 

e.g.

```
/// A `LogHandler` is an implementation of a logging backend. 
/// 
/// This type is an implementation detail and should not normally be accessed, unless implementing your own logging backend.
/// For user API of please refer to the documentation of `Logger`.
```

or something like that?